mindocr/data/utils/polygon_utils.py

```diff
@@ -0,0 +1,20 @@
+from typing import Union
+import numpy as np
+
+_2PI = 2 * np.pi
```

Comment:
it seems unnecessary to define _2PI globally

Reply:
This is a module local variable because it starts with `_`. The idea is to save computational time 😁.
If you try to access it from elsewhere, you'll fail:
![image](https://github.com/mindspore-lab/mindocr/assets/16683750/b4788fb3-e38b-43c5-a085-98f9374c0881)